packages/create-next-app/index.ts

```diff
@@ -21,7 +22,11 @@ const handleSigTerm = () => process.exit(0)
 process.on('SIGINT', handleSigTerm)
 process.on('SIGTERM', handleSigTerm)
 
-const onPromptState = (state: any) => {
+const onPromptState = (state: {
+  value: InitialReturnValue
+  aborted: boolean
+  exited: false
```

Comment:
```suggestion
  exited: boolean
```

Is this really always false or should this be a boolean?

Reply:
🤦

Thank you.